src/scss/components/_widget.scss

```diff
@@ -18,6 +18,13 @@
   }
 
   &__title {
+    @include media(small) {
```

Comment:
@eduardojmatos Much better way of including media queries, 💯 
Can we use this in other projects?

Reply:
@iondrimba yes, I think is ok to use this now. Scorpion project has some variables (breakSmall, etc.), because at begining of Gaiden import to him, we don't move the source code of Gaiden to vendor/gaiden folder.